<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>musicplayer</class>
 <widget class="QWidget" name="musicplayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>musicplayer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#musicplayer{background-image: url(:/image/background.jpg); background-repeat: no-repeat;background-position: center;}</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>256</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>298</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="file">
      <property name="minimumSize">
       <size>
        <width>56</width>
        <height>56</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16666666</width>
        <height>16666666</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/image/file.png)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="presong">
      <property name="minimumSize">
       <size>
        <width>56</width>
        <height>56</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16666666</width>
        <height>16666666</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/image/preSong.png)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="start">
      <property name="minimumSize">
       <size>
        <width>56</width>
        <height>56</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16666666</width>
        <height>16666666</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(:/image/start.png)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="nextsong">
      <property name="minimumSize">
       <size>
        <width>56</width>
        <height>56</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16666666</width>
        <height>16666666</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/image/nextSong.png)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sound">
      <property name="minimumSize">
       <size>
        <width>56</width>
        <height>56</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16666666</width>
        <height>16666666</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/image/volume.png)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="curLabel">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="totalLabel">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSlider" name="playCourseSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
